middlewares/studentAssessmentAccess.ejs

```diff
@@ -32,13 +32,13 @@
                       <div class="col-md-3 col-sm-6">
                         <%- include('../pages/partials/scorebar', {score: assessment_instance.score_perc}) %>
                       </div>
-                      <div class="col-md-6 col-sm-12 text-right">
-                          Assessment is <strong>closed</strong> and is no longer available
+                      <div class="col-md-6 col-sm-12 text-right assessment-closed-message">
```

Comment:
Where is this css class `assessment-closed-message` defined?  I don't see it anywhere in this PR or in master.

Reply:
It's not defined 😄

It's only used by the test suite. The tests use this class to make sure we are on the correct page and have the correct message.

Should we name it something else? `test-suite-assessment-closed-message`?

Reply:
Ooh, yeah that makes sense.  Changing it to that would definitely make this clearer.

Reply:
done